package/readme.txt

```diff
@@ -52,6 +52,7 @@ Bug fixes:
 - Restore file table configuration (column visibility, width and position) on startup also when custom folders are set.
 - The default keyboard shortcut for packing files (CTRL+I) opens the 'Pack files' dialog on macOS.
 - File search works with search strings that resemble a path to an archive file, e.g., find:///home/*.zip.
+- Improve the handling of edge cases when opening compressed archives and choosing a custom extension.
```

Comment:
this needs to be more informative or we should rather consider this part a code change and drop it - what do you think?

Reply:
Initially I added this as there are user-facing changes, some bugs were fixed and some things work better now. From the top of my head an example would be Open as -> tar.bz2 failing to work properly or the filename chosen not being very good when the file extension was wrong. However, I could not come up with a good summary and I don't like it much either so I'm for dropping it unless you can come up with something better.

Reply:
agree, lets drop it then

Reply:
Done